components/orbit-controls/examples/basic/index.html

```diff
@@ -4,13 +4,12 @@
     <meta charset="utf-8">
     <title>A-Frame Orbit Controls Component - Basic</title>
     <meta name="description" content="Basic example for A-Frame Orbit Controls component."></meta>
-    <script src="https://aframe.io/releases/0.8.2/aframe.min.js"></script>
+    <script src="https://aframe.io/releases/1.4.1/aframe.min.js"></script>
     <script src="../../dist/aframe-orbit-controls.min.js"></script>
-    <script src="https://unpkg.com/aframe-supercraft-loader@3.0.0/dist/aframe-supercraft-loader.js"></script>
   </head>
   <body>
     <a-scene>
-      <a-entity supercraft-loader="name: icky-snake"></a-entity>
+      <a-torus-knot color="#B84A39" arc="180" p="2" q="7" radius="5" radius-tubular="0.1"></a-torus-knot>
```

Comment:
Replaced the supercraft loader with a torus-knot. Supercraft loader no longer seems to work since aframe 1.1.0